src/OpenTelemetry/Trace/Configuration/OpenTelemetrySdk.cs

```diff
@@ -33,10 +33,11 @@ static OpenTelemetrySdk()
         /// Enables OpenTelemetry.
         /// </summary>
         /// <param name="configureOpenTelemetryBuilder">Function that configures OpenTelemetryBuilder.</param>
+        /// <returns><see cref="IDisposable"/> to be disposed on application shutdown.</returns>
         /// <remarks>
         /// Basic implementation only. Most logic from TracerBuilder will be ported here.
         /// </remarks>
-        public static void EnableOpenTelemetry(Action<OpenTelemetryBuilder> configureOpenTelemetryBuilder)
+        public static IDisposable EnableOpenTelemetry(Action<OpenTelemetryBuilder> configureOpenTelemetryBuilder)
```

Comment:
@cijothomas I couldn't make the unit tests work without a way to destroy the ActivityListener we create internally so I return it as an IDisposable here. But we'll also need to also stop any ActivityProcessors/ActivityExporters on application shutdown right?

Reply:
Yes, we should gracefully call processors & exporters to ensure they flush active spans before the application closes. I don't see a problem with making this `IDisposable` but we'll need to ensure docs and examples are updated to include this as a best practice too.